src/MainWindow.vala

```diff
@@ -306,6 +331,16 @@ public class AppCenter.MainWindow : Hdy.ApplicationWindow {
         search_entry.text = term;
     }
 
+    public void send_installed_toast (AppCenterCore.Package package) {
+        last_installed_package = package;
+
+        if (selected_package == null || (selected_package != null && selected_package.get_name () != package.get_name ())) {
```

Comment:
This probably deserves a comment about what this solves for. It's not immediately obvious to me what this conditional does

Reply:
Fair point

Reply:
Quite verbose... Open for feedback:

```
// Show a toast if
// no package is selected --> we are not on an app page
// or
// a package is selected --> we are on an app page
// but the app's name is not equal to the package's name that should be installed
if (selected_package == null || (selected_package != null && selected_package.get_name () != package.get_name ())) {
    toast.title = _("“%s” has been installed").printf (package.get_name ());

    toast.send_notification ();
}
```

Reply:
Suggestion added above